src/client/utils/validations.js

```diff
@@ -245,19 +281,19 @@ function isValidDeductibleExpenses(data) {
     isValidField(isValidMonetaryValue, data.deductibleEducationExpenses);
 }
 
-function isValidAdditionalInformation(data) {
+function isValidVAFacility(data) {
   return validateIfDirty(data.facilityState, isNotBlank) &&
     validateIfDirty(data.vaMedicalFacility, isNotBlank);
 }
 
 function isValidMedicareMedicaid(data) {
-  return isBlankDateField(data.medicarePartAEffectiveDate) ||
-    isValidDateField(data.medicarePartAEffectiveDate);
+  return validateIfDirty(data.isMedicaidEligible, isNotBlank) &&
+    validateIfDirty(data.isEnrolledMedicarePartA, isNotBlank) &&
```

Comment:
Does `isEnrolledMedicarePartA` need to be required if my answer to `isMedicaidEligible` is "no"?